.github/dependabot.yml

```diff
@@ -108,6 +108,9 @@ updates:
       # Logstash
       - dependency-name: biz.paluch.logging:logstash-gelf
       - dependency-name: io.sentry:*
+      # MongoDB
+      - dependency-name:  org.mongodb:mongodb-driver-core
+      - dependency-name:  org.mongodb:mongodb-crypt
```

Comment:
Just add a wildcard for `org.mongodb:*`, that will be more future-proof.

Reply:
Done